tests/Mailpit.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { test, expect } from '../src/index';
+
+test('Verify sign-up email', { tag: '@Email' }, async ({
+    ShopCustomer,
+    StorefrontAccountLogin,
+    StorefrontAccount, IdProvider,
+    page,
+    MailpitApiContext,
+    TestDataService,
+}) => {
+    // Delete previous mails
+    await MailpitApiContext.deleteMail();
```

Comment:
this will also delete email of other tests, which will make it fail.

Reply:
removed, test creates somehow two customers one with admin name and one with John Goldblum. But I was able to assert my customer's email, even not deleting previous mails.